### PR TITLE
scripts: Add scripts for building `acond` on `rust` Alpine containers

### PR DESCRIPTION
**Pull request by binxing**
This PR contains scripts for building `acond` reproducibly on Alpine. 

Documentation on those scripts will come soon.

### CODE REVIEW

File: acond/README.md [hunk 1]
```diff
@@ -1,60 +1,192 @@
...
+Executing busybox-1.36.1-r2.trigger
+OK: 201 MiB in 38 packages
+/acon/acond $
+```
+
+Please note `start_rust_buildenv` identifies (using `git rev-parse --show-toplevel`) and maps `path/to/ACON_PROJECT_ROOT` to `/acon` inside the container, and sets the working directory to `/acon/acond` automatically. Therefore, building `acond` will be as simple as below.
+
+```sh
+./build
+```
```
Comment (xiangquanliu) [inline acond/README.md:74]: Should '$' be added before the command line? like -

$ ./build

File: acond/README.md [hunk 2]
```diff
@@ -1,60 +1,192 @@
...
+
+   ```sh
+   ./build_static -r -Ffull
+   ```
+
+#### Reproducing Build Environment
+
+`start_rust_buildenv` keeps in `/path/to/ACON_PROJECT_ROOT/scripts/dep/rust-VERSION/OS/` copies of all packages (e.g., *.apk files for Alpine Linux) installed in the container, along with a `PACKAGES-INFO` file recording the OS version and date/time when those packages were downloaded. Those files, along with a proper [`rust`][rust-docker] tag, can be used to reproduce the build environment.
+
+For example, given `scripts/deps/rust-1.72.0/alpine/PACKAGES-INFO` and the `*.apk` files, the commands below reproduces the build environment.
```
Comment (xiangquanliu) [inline acond/README.md:121]: reproduces -> reproduce?

Comment (binxing) [reply acond/README.md:121]: Good catch! I'll fix.

File: acond/README.md (hunk 1 above)
Comment (binxing) [reply acond/README.md:74]: I'm not sure if it's a widely adopted convention, but I put in the prompt string only if what follows is the output of some program. In this case it's typed in by the user so I don't put in `$` here.

**Pull request merged**